globus_action_provider_tools/testing/mocks.py

```diff
@@ -1,45 +1,41 @@
-from typing import NamedTuple
-from unittest.mock import PropertyMock, patch
+from unittest.mock import Mock
```

Comment:
So, here, unittest.mock makes sense to me as it doesn't really on any outside packages to be installed so people not going flask-y can also not go pytest-y.

Either I'm over thinking this, or under thinking it. I'm not quite sure...

Reply:
I'm not sure how to prove that it's over or under thinking (: My hope in providing base mocks was that they'd be able to be composed by higher level tools depending on what needed to be accomplished. I think it worked nicely that the `fixtures` end up dogfooding the `mocks` and that the `tokenchecker` mock dogfoods the `auth_state` mock. From some of my udpates to the examples, they also seem to fit in well when using `monkeypatch`.